lib/ui/src/modules/ui/components/layout/index.js

```diff
@@ -140,14 +140,6 @@ class Layout extends React.Component {
     this.onDragEnd = this.onDragEnd.bind(this);
   }
 
-  componentDidMount() {
-    window.addEventListener('resize', this.onResize);
```

Comment:
are these not needed anymore?

I think we still need to handle the case when the browser window gets resized?

Reply:
I don’t think so. Because `onResize` returns another function the window resize events never actually updates the state. 

I will have to double check but I think the `SplitPane` `onChange` handler fires when browser window is resizes.

Reply:
Yes we need. Try to resize window here https://deploy-preview-3180--storybooks-official.netlify.com

Reply:
@Hypnosphi I also doubled checked on `master` resizing the browser window never triggered `onResize`

Reply:
> I also doubled checked on `master` resizing the browser window never triggered `onResize`

Oh I see. Looks like a bug worth fixing to me

Reply:
We should probably extract the part  that updates `previewPanelDimensions` from `onResize`, because it's the only thing that should be done on window resize

Reply:
Sounds good!. I'll change that in this PR!